OneSignalSDK/onesignal/src/main/java/com/onesignal/LocationGMS.java

```diff
@@ -85,51 +86,82 @@
       void complete(LocationPoint point);
    }
 
+   abstract static class LocationAndPromptHandler implements LocationHandler {
```

Comment:
Can you explain what you mean by `Location And PromptHandler`? Can we call this `LocationPromptCompletionHandler`?

Reply:
actually yeah 😂